src/commands/update.js

```diff
@@ -28,7 +29,7 @@ const fetchCrontab = async () => {
 }
 
 const isJob = ( line ) => {
-  const schedule = line.slice(0,10)
+  const { schedule } = parse(line);
   return cron.validate(schedule);
 }
 
```

Comment:
I'm not sure we need all of that parsing to only get the schedule, this function splits the whole array, then does two joins and two slices when all we need is the first 10 chars. Might not make a huge difference but its for every line.

Reply:
I think using the `parse` command might be a little overkill in that it slices and joins the `command`, but I think we do need to split, slice and join the line to account for the different schedule possibilities. Schedules like `*/5 * 1/4 8,9,12 *` can have varying lengths of greater than 10 chars so to be read properly for validation they'd need to take the first 5 fields of the line.

Reply:
I'll tweak it to avoid the additional `command` parsing